package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "systelab-components-wdio-test",
-  "version": "1.7.0",
```

Comment:
@Franek75, I reckon latest published version of lib was 1.16.6 so it needs to be updated to 1.16.7 but not in package-lock.json file but in package.json one

Reply:
Thank you. package-lock.json update reverted, and package.json updated as suggested